test/integration_test/stork-specs/stork-deployment.yaml

```diff
@@ -0,0 +1,237 @@
+        name: VolumeBinding
+      plugins:
+        bind: {}
+        filter: {}
+        multiPoint:
+          enabled:
+          - name: PrioritySort
+          - name: NodeUnschedulable
+          - name: NodeName
+          - name: TaintToleration
+            weight: 3
+          - name: NodeAffinity
+            weight: 2
+          - name: NodePorts
+          - name: NodeResourcesFit
+            weight: 1
+          - name: VolumeRestrictions
+          - name: EBSLimits
+          - name: GCEPDLimits
+          - name: NodeVolumeLimits
+          - name: AzureDiskLimits
+          - name: VolumeBinding
+          - name: VolumeZone
+          - name: PodTopologySpread
+            weight: 2
+          - name: InterPodAffinity
+            weight: 2
+          - name: DefaultPreemption
+          - name: NodeResourcesBalancedAllocation
+            weight: 1
+          - name: ImageLocality
+            weight: 1
+          - name: DefaultBinder
+        permit: {}
+        postBind: {}
+        postFilter: {}
+        preBind: {}
+        preFilter: {}
+        preScore: {}
+        queueSort: {}
+        reserve: {}
+        score: {}
+      schedulerName: stork
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: stork-account
+  namespace: kube-system
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+   name: stork-role
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: stork-role-binding
+subjects:
+- kind: ServiceAccount
+  name: stork-account
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: stork-role
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: stork-service
+  namespace: kube-system
+spec:
+  selector:
+    name: stork
+  ports:
+    - name: extender
+      protocol: TCP
+      port: 8099
+      targetPort: 8099
+    - name: webhook
+      protocol: TCP
+      port: 443
+      targetPort: 443
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  labels:
+    tier: control-plane
+  name: stork
+  namespace: kube-system
+spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      name: stork
+  replicas: 3
+  template:
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+      labels:
+        name: stork
+        tier: control-plane
+    spec:
+      containers:
+      - command:
+        - /stork
+        - --driver=pxd
+        - --verbose
+        - --leader-elect=true
+        # Uncomment the line below if you want to enable the feature to
+        # automatically update schedulerName
+        #- --app-initializer=true
+        imagePullPolicy: Always
+        image: openstorage/stork:2.2.4
```

Comment:
I am not getting the motivation for adding these specs, even the deprecated one. Also the version in this is pretty old.

Reply:
Actually the entire `stork-specs` folder was added to the PR by mistake. Will delete it.